elb.tf

```
@@ -0,0 +1,81 @@
+resource "aws_security_group" "elb" {
+  name_prefix = "${var.name}-elb-sg"
```

`sg` is redundant since it's clear this is a security group. Also more specificity should be loaded onto the front, not the back

```suggestion
  name_prefix = "elb-${var.name}"
```